aioredisqueue/queue.py

```diff
@@ -160,12 +170,18 @@ def _ack(self, task_id, method='multi'):
             args=[task_id],
         )
 
-    async def _requeue(self, before=None):
+    async def _requeue(self, now, before=-1):
         if 'requeue' not in self._lua_sha:
             await self._load_scripts('requeue')
 
         return await self._redis.evalsha(
```

Comment:
it is probably better to first get the last timestamp from redis and call lua only if no one else haven't already done with requeueing.